gradle-plugins/biz.aQute.bnd.gradle/src/main/java/aQute/bnd/gradle/BndPlugin.java

```diff
@@ -902,6 +905,32 @@ public void execute(Task tt) {
 		}
 	}
 
+	/**
+	 * Adds an {@link ArtifactView} to the given compile task's classpath that will explicitly request the default `jar`
+	 * variant of the given configuration possessing the {@link LibraryElements#JAR} attribute.
+	 *
+	 * This ensures that even if Gradle uses the `classes` variant of the configuration which uses the {@link LibraryElements#CLASSES}
+	 * attribute, the `jar` variant will also be available on the classpath, making any resources packaged in the jar
+	 * readable.  The provided configuration itself will <strong>NOT</strong> be modified.
+	 *
+	 * @param compileClasspath the compile classpath configuration to use as a basis for an {@link ArtifactView}
+     * @param task the compile task whose classpath will be modified
+	 */
+	private void addLibsToClasspath(Configuration compileClasspath, AbstractCompile task) {
+		ArtifactView librariesView = compileClasspath.getIncoming().artifactView(vc -> {
+			AttributeContainer attributes = vc.getAttributes();
+			try {
+				attributes.attribute(LibraryElements.LIBRARY_ELEMENTS_ATTRIBUTE, project.getObjects().named(LibraryElements.class, LibraryElements.JAR));
+				task.getLogger()
+						.info("Added libraries jars to to {}:{}", project.getPath(), compileClasspath.getName());
+			} catch (IllegalArgumentException e) {
+				task.getLogger()
+						.info("Unable to add libraries jars to to {}:{}", project.getPath(), compileClasspath.getName(), e);
+			}
+		});
+		task.setClasspath(task.getClasspath().plus(librariesView.getFiles()));
```

Comment:
We shouldn't this.  IIUC, the jar produced by projects using BndPlugin is different from just the class files, so the whole point is to only have the jar.

If that's the case, then we can just leave the method